app/models/chunk.rb

```diff
@@ -44,31 +44,21 @@ def sub_path
   end
 
   def expiry
```

Comment:
Since the StorageProvider now handles building the signed url, is `Chunk#expiry` still being used anywhere?

Reply:
removed